shanoir-ng-datasets/src/main/java/org/shanoir/ng/studycard/controler/QualityCardApiController.java

```diff
@@ -169,4 +176,18 @@ public ResponseEntity<QualityCardResult> testQualityCardOnStudy(
         return new ResponseEntity<>(results, HttpStatus.OK);
     }
 
+	@Override
+    public ResponseEntity<QualityCardResult> testQualityCardOnStudy(
+        @Parameter(name = "id of the quality card", required = true) @PathVariable("qualityCardId") Long qualityCardId,
+		@Parameter(name = "dataset number start ", required = true) @PathVariable("start") int start,
```

Comment:
Hi, please change swagger here from "dataset number start" to "examination number start". Thank you!